### PR TITLE
Import scripts 20191119

### PR DESCRIPTION
**Pull request by GeoWill**

### CODE REVIEW

File: polling_stations/apps/data_collection/management/commands/import_west_oxfordshire.py [hunk 1]
```diff
@@ -3,15 +3,46 @@
...
 
     def address_record_to_dict(self, record):
         rec = super().address_record_to_dict(record)
         uprn = record.property_urn.strip().lstrip("0")
 
+        # if uprn == "10094864970":  # Annexe At Laurel Mount, 4 North Street, Middle Barton, Chipping Norton, Oxon
+        #     print(record)
+
+        #     record._replace(addressline6="OX7 7BJ")
+        #     print(record)
```
Comment (chris48s) [inline polling_stations/apps/data_collection/management/commands/import_west_oxfordshire.py]: there's a few bits of debug code to clean up here

File: polling_stations/apps/data_collection/management/commands/import_wirral.py [hunk 2]
```diff
@@ -40,24 +46,17 @@ def address_record_to_dict(self, record):
...
             "42072402",  # CH434TS -> CH431TS : 29A Mather Road, Oxton, Wirral
             "42072403",  # CH434TS -> CH431TS : 29B Mather Road, Oxton, Wirral
             "42081451",  # CH439TT -> CH439UE : Wexford Ridge, Noctorum Lane, Oxton, Wirral
+            "42181578",  # CH474AU -> CH472DH : 10 Market Street, Hoylake, Wirral
         ]:
             rec["accept_suggestion"] = True
 
         if uprn in [
-            "42026495",  # CH412SQ -> CH416AZ : 30 Clifton Road, Tranmere, Wirral
+            #             "42026495",  # CH412SQ -> CH416AZ : 30 Clifton Road, Tranmere, Wirral
```
Comment (chris48s) [inline polling_stations/apps/data_collection/management/commands/import_wirral.py]: is this one in or out?

Comment (GeoWill) [reply polling_stations/apps/data_collection/management/commands/import_wirral.py]: Out :slightly_smiling_face:

File: polling_stations/apps/data_collection/management/commands/import_west_oxfordshire.py (hunk 1 above)
Comment (GeoWill) [reply polling_stations/apps/data_collection/management/commands/import_west_oxfordshire.py]: :+1:

**Comment by GeoWill** [comment]
Fixed.

**Pull request merged**